backend/src/apiserver/main.go

```diff
@@ -73,6 +77,11 @@ func main() {
 	}
 
 	clientManager := cm.NewClientManager()
+	if common.IsOnlyKubernetesWebhookMode() {
+		startWebhookHTTPProxy(clientManager.ControllerClient())
```

Comment:
Could you please add a log message just before this call indicating it's in webhook only mode?

Reply:
Ah, nvm. I see you have a log message in the function. Nice!